bigtable_rs/src/bigtable.rs

```diff
@@ -86,10 +86,11 @@
 //! }
 //! ```
 
+use core::time;
 use std::sync::Arc;
 use std::time::Duration;
 
-use gcp_auth::AuthenticationManager;
+use gcp_auth::{AuthenticationManager, CustomServiceAccount};
```

Comment:
CustomServiceAccount should not be needed now I guess?

Reply:
Removed. Thanks for the heads up